### ISSUE TITLE
Please add support for A/C (ir device)

### ISSUE THREAD

**Issue by amanolo** [OpenWonderLabs/homebridge-switchbot-openapi#70]
**Is your feature request related to a problem? Please describe.**
[SwitchBot] Unable to retreive device status.

**Describe the solution you'd like**
Please add support for A/C (ir device)

**Describe alternatives you've considered**

**Additional context**

```
{
	"statusCode":100,
	"body":	{
		"deviceList": [
			],
			"infraredRemoteList": [
					{
						"deviceId": "02-202101201235-11403291",
						"deviceName": "Living Room Air Conditioner",
						"remoteType":"Air Conditioner",
						"hubDeviceId":"EEDA097901BD"
					}
			]
	},
	"message":"success"
}
```

**Comment by donavanbecker** [comment]
duplicate of #55

**Comment by donavanbecker** [comment]
As you probably already know, this has been released with v2.1.0
